documentation/src/docs/segment-store-service.md

```diff
@@ -66,117 +96,116 @@ Segment Containers are a logical grouping of Segments, and are responsible for a
 - **Durable Log**: The Container writes every operation it receives to this log and acks back only when the log says it has been accepted and durably persisted.
 - **Read Index**: An in-memory index of where data can be read from. The Container delegates all read requests to it, and it is responsible for fetching the data from wherever it is currently located (Cache, Tier 1 Storage or Tier 2 Storage).
 - **Cache**: Used to store data for appends that exist in Tier 1 only (not yet in Tier 2), as well as blocks of data that support reads.
-- **Storage Writer**: Processes the durable log operations and applies them to Tier 2 storage (in the order in which they were received). This component is also the one that coalesces multiple operations together, for better back-end throughput.
+- **Storage Writer**: Processes the durable log operations and applies them to Tier 2 Storage (in the order in which they were received). This component is also the one that coalesces multiple operations together, for better back-end throughput.
 
 ### Segment Container Metadata
-The Segment Container Metadata is critical to the good functioning and synchronization of its components. This metadata is shared across all components and it comes at two levels: "Container-wide Metadata" and "per-Segment Metadata". Each serves a different purpose and is described below.
+The Segment Container Metadata is critical to the good functioning and synchronization of its components. This metadata is shared across all components and it comes at two levels: "Container-wide metadata" and "per-Segment metadata". Each serves a different purpose and described below.
```

Comment:
Add "is" back. `is described` is how it should be.

Reply:
Reverted.(Accepted Suggestion)